src/routes/iofog.js

```diff
@@ -306,6 +306,39 @@ module.exports = [
       const getHalUsbInfo = ResponseDecorator.handleErrors(FogController.getHalUsbInfoEndPoint, successCode, errCodes)
       const responseObject = await getHalUsbInfo(req)
 
+      res
+        .status(responseObject.code)
+        .send(responseObject.body)
+
+      logger.apiRes({ req: req, res: responseObject })
+    }
+  },
+  {
+    method: 'post',
+    path: '/api/v3/iofog/:uuid/prune',
+    middleware: async (req, res) => {
+      logger.apiReq(req)
+
+      const successCode = constants.HTTP_CODE_NO_CONTENT
+      const errCodes = [
+        {
+          code: 400,
+          errors: [Errors.ValidationError]
+        },
+        {
+          code: 401,
+          errors: [Errors.AuthenticationError]
+        },
+        {
+          code: 404,
+          errors: [Errors.NotFoundError]
+        }
+      ]
+
+      const setFogRebootCommand = ResponseDecorator.handleErrors(FogController.setFogPruneCommandEndPoint,
```

Comment:
`setFogRebootCommand` That's a copy/paste artifact. Please update the name to reflect what the var is used for.

Reply:
updated